baseband/vlbi_base/tests/test_vlbi_base.py

```diff
@@ -430,7 +431,7 @@ def test_streambase_squeeze(self):
                                  'n1, n2, n3, n4, n5, n6, n7, n8')
         sample_shape = smp_shp_cls(1, 17, 3, 2, 1, 5, 1, 1)
         sb = VLBIStreamBase(None, None, sample_shape, 1, False,
```

Comment:
Looking at this test again, I think we should use keyword arguments -- I've got no idea what it is doing!!